CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.10.0
+
+* Deprecated `dropRequestAfter` in favor of more precise `readTimeout`
```

Comment:
Why deprecate `dropRequestAfter` btw, wouldn't that be nice to still have real max timing?

Reply:
That is actually a good question. Probably makes sense to keep it as it can protect against "slow read" problems where endpoint writes super tiny chunks thus never triggering `readTimeout` if I understand it correctly.

Reply:
@DeTeam Added the change, could you please re+1?